compose.yaml

```
@@ -0,0 +1,18 @@
+services:
+  elorachat:
```

Nitpick: Add dash (-) here.
```suggestion
  elora-chat:
```